app/views/upload.scala.html

```diff
@@ -116,7 +116,7 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
             </div>
         </div>
         <div>
-            <progress class="progress-display" value="" max="100"></progress>
+            <progress class="progress-display" value="" max="100" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow=""></progress>
```

Comment:
Have you talked to Loki about the aria attributes we should be using on the upload progress bar? She's spent so much time on the file check page that I think she'll have some good ideas about what to do here.

But we'll do the accessibility testing for this page separately, so I think we should only make changes to it now if they're obviously better! We don't want to spend too much time on this now if it makes it harder to wrap up the transfer summary work. 🙂

Reply:
Ah no, I hadn't spoken to her about them, I was just learning more about ARIA roles as part of Learning day (since the day before, I was looking at the ARIA for the transfer summary), decided to look at how we implemented them and came across this one and thought it should have this and after trying it out, it worked. Ok, I'll leave it as is and whoever tests it can either remove, keep or improve on it :).

Reply:
The HTML `progress` element is semantic and has the ARIA `progressbar` role implicitly, so we shouldn't need to add the `role="progressbar"` role. Not 100% sure about where we stand with the `aria-value` attributes off the top of my head though 👍

Reply:
Ah yeah, the `progressbar` comment makes sense; I'll remove it, thanks 👍